libs/libvtrutil/src/vtr_vector.h

```diff
@@ -61,7 +61,6 @@ class vector : private std::vector<V, Allocator> {
     using storage::size;
 
     using storage::back;
-    using storage::data;
```

Comment:
Fixed that in #1282 (it was also needed to then actually provide an implementation of data() for all the places that actually use `data()` for non-`vector<bool>` specialized vectors)

Reply:
Thanks!